### ISSUE TITLE
PEATIO VERSION V3 - COMING SOON !

### ISSUE THREAD

**Issue by algobasket** [algobasket/PeatioCryptoExchange#32]
New Multiple Templates
All New Trading Pages
Easy Coin Installation 
Robust Admin Backoffice
More Security

**Comment by toboho** [comment]
About what time

**Comment by kodcoin** [comment]
Why can not you contact me?Bithumb-Clone-Script Contact me

**Comment by algobasket** [comment]
This Month

**Comment by samqju** [comment]
Looking forward to V 3.0, I tried installing version 2.0 and was successful but email verification and sms didn't worked

**Comment by bkkite** [comment]
Will be compatible with cryptonote coins?

**Comment by algobasket** [comment]
> Will be compatible with cryptonote coins?
Any Coin from this list : http://coinmarketrise.com/index.php/crawl/getCoinMarketCapData

**Comment by faneaatiku** [comment]
> > Will be compatible with cryptonote coins?
> > Any Coin from this list : http://coinmarketrise.com/index.php/crawl/getCoinMarketCapData

What programming language do you use on v3 ? Keeping ruby or going with something else ?

**Comment by algobasket** [comment]
Ruby version will also be there but we are making it available on different technologies mainly Python, PHP, Nodejs and .NET . Fully Restful

**Comment by faneaatiku** [comment]
> Ruby version will also be there but we are making it available on different technologies mainly Python, PHP, Nodejs and .NET . Fully Restful

Interesting. So you plan on creating the entire backend in all major programming languages ? 
I see only Pusher is supported. Any plans on creating a websocket implementation?

**Comment by algobasket** [comment]
> > Ruby version will also be there but we are making it available on different technologies mainly Python, PHP, Nodejs and .NET . Fully Restful
> 
> Interesting. So you plan on creating the entire backend in all major programming languages ?
> I see only Pusher is supported. Any plans on creating a websocket implementation?

Pusher is paid version for web socket with less code implementation . But websocket we can build it own without using pusher easily with socketjs + nodejs or other listening tools running as micro services inside a container .

**Comment by faneaatiku** [comment]
> > > Ruby version will also be there but we are making it available on different technologies mainly Python, PHP, Nodejs and .NET . Fully Restful
> > 
> > 
> > Interesting. So you plan on creating the entire backend in all major programming languages ?
> > I see only Pusher is supported. Any plans on creating a websocket implementation?
> 
> Pusher is paid version for web socket with less code implementation . But websocket we can build it own without using pusher easily with socketjs + nodejs or other listening tools running as micro services inside a container .

Yes, since rabbitmq is already a requirement, a simple socket would resolve it. Thanks for answering. 

Let's see the PHP/Python/Nodejs up and maybe . get some free time to help with some development. Keep up the good work 👍

**Comment by bkkite** [comment]
That sounds, great!! Looking forward to see this new release 👍

**Comment by not-a-dev0** [comment]
When v3?

**Comment by bkkite** [comment]
any news about when? ;)

**Comment by zutugecof** [comment]
keep it up doing great work ! Please lemme know when you will release it 
Thanks

**Comment by zaprex** [comment]
algobasket is bullshit dont use this shity vuln script

**Comment by bjay27** [comment]
Don't use algobasket version itw vulnerable

**Comment by TTcoinDeveloper** [comment]
It seems soo

**Comment by zaprex** [comment]
hahahhaha you ashole algobasket why you delete my comments dont use this shit vuln script

**Comment by bkkite** [comment]
Sorry but your message does not contribute anything, if you think there is
a vulnerability. Write what and so someone can solve it. but your message
is only meant to troll.

**Comment by algobasket** [comment]
@bkkite Peatio v2 is old with old deprecated gem file some undefined functions detect as vulnerable , so you can fix it yourself by updating the specific gem packages . Peatio v3 coming soon as it will updated code and its still under development and testing phase .

**Comment by cipherzzz** [comment]
@algobasket - I am really excited about v3. I can help with the node.js code if you need help... send me a message if so. Thanks!

**Comment by bkkite** [comment]
Hi, @algobasket.  What about v3?

**Comment by blacklevel7** [comment]
> New Multiple Templates
> All New Trading Pages
> Easy Coin Installation
> Robust Admin Backoffice
> More Security

We very very look forward to the upgrade . At least UI.

**Comment by algobasket** [comment]
> > New Multiple Templates
> > All New Trading Pages
> > Easy Coin Installation
> > Robust Admin Backoffice
> > More Security
> 
> We very very look forward to the upgrade . At least UI.

![screencapture-localhost-3000-2019-04-17-23_55_51](https://user-images.githubusercontent.com/8275673/56311785-6c216180-616c-11e9-8021-a7f86723962a.png)

**Comment by blacklevel7** [comment]
WOW! :)

**Comment by toboho** [comment]
good job!

**Comment by Apollo-IT** [comment]
Hi, @bkkite @toboho @kodcoin 
How are you?
You look like you're finding an open source cryptocurrency exchange.
I can guide you.
Please contact me.
Regards.

Skype: live:apollo447722
Email: apollo447722@outlook.com

**Comment by cryptohood** [comment]
> > > New Multiple Templates
> > > All New Trading Pages
> > > Easy Coin Installation
> > > Robust Admin Backoffice
> > > More Security
> > 
> > 
> > We very very look forward to the upgrade . At least UI.
> 
> ![screencapture-localhost-3000-2019-04-17-23_55_51](https://user-images.githubusercontent.com/8275673/56311785-6c216180-616c-11e9-8021-a7f86723962a.png)

when is release mate? am i better off to wait then edit the current version as a launched project? are you able to update from current v2 - v3 ?

cheers

**Comment by algobasket** [comment]
> > > > New Multiple Templates
> > > > All New Trading Pages
> > > > Easy Coin Installation
> > > > Robust Admin Backoffice
> > > > More Security
> > > 
> > > 
> > > We very very look forward to the upgrade . At least UI.
> > 
> > 
> > ![screencapture-localhost-3000-2019-04-17-23_55_51](https://user-images.githubusercontent.com/8275673/56311785-6c216180-616c-11e9-8021-a7f86723962a.png)
> 
> when is release mate? am i better off to wait then edit the current version as a launched project? are you able to update from current v2 - v3 ?
> 
> cheers

OK NETPOL .. !

**Comment by blacklevel7** [comment]
> New Multiple Templates
> All New Trading Pages
> Easy Coin Installation
> Robust Admin Backoffice
> More Security

Hi @algobasket 
Maybe there is some news? :)

**Comment by wes45** [comment]
Is this announced update for real, or is it to keep the thread lively? 6 months have passed and no update whatsoever, also no fixes for the bugs in the previous version. That is a pity

**Comment by URSTYLE** [comment]
Any news or topic is dead?

**Comment by algobasket** [comment]
> Any news or topic is dead?
Its ready !

**Comment by LightBurdenOfficial** [comment]
> > Any news or topic is dead?
> > Its ready !

Where are you ready?

**Comment by wahaobi** [comment]
> > 任何新闻或话题都死了？
> > 准备好了 ！

好了没。

**Comment by pblab-dev** [comment]
It's hard to find information, I suggest you see this https://bitcointalk.org/index.php?topic=5202005.new#new

**Comment by cheezcharmer** [comment]
list of things that never happened

**Comment by francissantiago** [comment]
https://github.com/francissantiago/peatio-spero-exchange

**Comment by kripanidhi** [comment]
is this project active ?

**Comment by algobasket** [comment]
> is this project active ?

its on hold for new version but we will release it soon

**Comment by pwz2k** [comment]
Updates on v3 releasing?

**Comment by algobasket** [comment]
> Updates on v3 releasing?

Yes soon
